build.sbt

```diff
@@ -16,15 +16,9 @@
 
 name := "hyperspace-core"
 
-sparkVersion := "2.4.2"
+sparkVersion := "3.0.0"
 
-lazy val scala212 = "2.12.8"
-lazy val scala211 = "2.11.12"
-lazy val supportedScalaVersions = List(scala212, scala211)
-
-scalaVersion := scala212
-
-crossScalaVersions := supportedScalaVersions
+scalaVersion := "2.12.10"
```

Comment:
Does it mean that switching to Spark 3.0, Hyperspace only runs on Scala 2.12?

Reply:
Yes. Spark 3.0 is compiled only with Scala 2.12.